### PR TITLE
clarify rspec instructions

### PR DESCRIPTION
**Pull request by ignaciosy**
- rexml gem is needed starting from ruby 3.0
- Improve test logging
- Clarify instructions

### CODE REVIEW

**Comment by schmijos** [comment]
> rexml gem is needed starting from ruby 3.0

Can you elaborate on that? Why is it needed?

**Comment by rnestler** [comment]
The rexml gem comes bundled with ruby > 3.0 so it doesn't make sense to add it, no?

File: ruby_on_rails/rspec.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Add the following gems to your Gemfile:
 
 ```ruby
 group :development, :test do
+  gem 'rexml'
```
[the remark this answers is not in the document]
Comment (schmijos) [reply ruby_on_rails/rspec.md:9]: To address the issue again: `rexml` was removed from the standard gems per Ruby 3.0

Are you sure about the requirement? As far as I can see, this is rather related to [activesupport](https://github.com/rails/rails/pull/40281/files).

**Comment by schmijos** [comment]
Ah, I just saw, that this PR was very old. We shouldn't have merged it.

**Pull request merged**